packages/test_packages_common.py

```diff
@@ -16,6 +20,22 @@ def registered_packages():
     return packages
 
 
+@functools.cache
+def built_packages() -> List[str]:
+    """Returns a list of built package names"""
```

Comment:
Comment added. It's not consistent but also not exactly the same thing,
 1. when building we need to know if build artifacts in packages/*/ folders were created (with `.packaged`) and so the package doesn't need to be rebuilt
 2. here we want to know if the final build artifacts exist

For instance, it could happen that the build/ folder was removed (meaning tests cannot be run) but the `pacakges/**/.packaged` are still there and packages don't need to be rebuilt from 0.